Tests/LibWeb/TestConfig.ini

```diff
@@ -20,6 +20,7 @@ Text/input/css/transition-basics.html
 ; Flaky on CI, see #19
 Text/input/WebAnimations/misc/DocumentTimeline.html
 Text/input/WebAnimations/animation-properties/currentTime.html
+Text/input/WebAnimations/animation-methods/updatePlaybackRate.html
```

Comment:
I find this very suspicious, have we seen this flake before?

This ties into `requestAnimationFrame`, which eventually hits `Navigable::has_a_rendering_opportunity` followed by `browsing_context->page().client().is_ready_to_paint()` which is something that you change in this PR.

Reply:
right, I disabled it because it's timing-sensitive, like some other animation tests we had to disable, and started failing on CI for this PR because now we go slightly faster and things are executed in different order. so it's this rare case when the new change is ok, but test we had is bad.

Reply:
That definitely needs to go in the commit message then :)